tex.rst

```diff
@@ -1190,13 +1190,11 @@ B
 
 .. code-block:: latex
 
-    \Bbb                    mediawiki-texvc
```

Comment:
`\Bbb` is in `mediawiki-texvc, but it is also in the core TeX input jax.  So I don't think this should be removed (as it does exist), but I could see removing the `mediawiki-texvc` entry.

Reply:
It's just moved lower but duplicated there.

Maybe it should be removed from `mediawiki-texvc.js` in general?

Reply:
Argh!  I didn't see that.  I did a pretty poor job putting this list together originally, it seems.

Reply:
It probably should be removed from `mediawiki-texcv`.